doc/sphinx/proofs/writing-proofs/reasoning-inductives.rst

```diff
@@ -5,269 +5,291 @@ Reasoning with inductive types
-         where constructor is executed and not in the context where :tacn:`apply`
-         is executed (the introductions are not taken into account).
-
-   .. tacv:: split {? with @bindings }
-      :name: split
-
-      This applies only if :g:`I` has a single constructor. It is then
-      equivalent to :n:`constructor 1 {? with @bindings }`. It is
-      typically used in the case of a conjunction :math:`A \wedge B`.
+   .. exn: The type has no constructors.
+      :undocumented:
 
-      .. tacv:: exists @bindings
-         :name: exists
+.. tacn:: split {? with @bindings }
 
-         This applies only if :g:`I` has a single constructor. It is then equivalent
-         to :n:`intros; constructor 1 with @bindings.` It is typically used in
-         the case of an existential quantification :math:`\exists x, P(x).`
+   Equivalent to :n:`constructor 1 {? with @bindings }` when the
+   conclusion is an inductive type with a single constructor.  The :n:`@bindings`
+   specify any parameters required for the constructor. It is
+   typically used to split conjunctions in the conclusion such as `A /\\ B` into
+   two new goals `A` and `B`.
 
-      .. tacv:: exists {+, @bindings }
+.. tacn:: exists {*, @bindings }
 
-         This iteratively applies :n:`exists @bindings`.
+   Equivalent to :n:`intros` followed by :n:`constructor 1 {? with binding__i }`
+   for each binding when the conclusion is an
+   inductive type with a single constructor.  It is typically used on
+   existential quantifications in the form `exists x, P x.`
 
-      .. exn:: Not an inductive goal with 1 constructor.
-         :undocumented:
+.. exn:: Not an inductive goal with 1 constructor.
+   :undocumented:
 
-   .. tacv:: left {? with @bindings }
-             right {? with @bindings }
-      :name: left; right
+.. tacn:: left {? with @bindings }
+          right {? with @bindings }
 
-      These tactics apply only if :g:`I` has two constructors, for
-      instance in the case of a disjunction :math:`A \vee B`.
-      Then, they are respectively equivalent to
-      :n:`constructor 1 {? with @bindings }` and
-      :n:`constructor 2 {? with @bindings }`.
+   These tactics apply only if :g:`I` has two constructors, for
+   instance in the case of a disjunction `A \\/ B`.
+   Then they are respectively equivalent to
+   :n:`constructor 1 {? with @bindings }` and
+   :n:`constructor 2 {? with @bindings }`.
 
-      .. exn:: Not an inductive goal with 2 constructors.
-         :undocumented:
+   .. exn:: Not an inductive goal with 2 constructors.
+      :undocumented:
 
-   .. tacv:: econstructor
-             eexists
-             esplit
-             eleft
-             eright
-      :name: econstructor; eexists; esplit; eleft; eright
+   .. tacn:: econstructor {? @nat_or_var {? with @bindings } }
+             eexists {*, @bindings }
+             esplit {? with @bindings }
+             eleft {? with @bindings }
+             eright {? with @bindings }
```

Comment:
Remove the indentation here, otherwise it looks like these are variants of `left` and `right`.